src/main/java/org/jitsi/jicofo/ParticipantChannelAllocator.java

```diff
@@ -363,7 +397,8 @@ else if (reInvite)
             }
             // Existing peers SSRCs
             RtpDescriptionPacketExtension rtpDescPe
-                = JingleUtils.getRtpDescription(cpe);
+                = net.java.sip.communicator.impl.protocol.jabber.jinglesdp
+                    .JingleUtils.getRtpDescription(cpe);
```

Comment:
i wonder if it's worth renaming this to `JicofoJingleUtils` to 1) avoid the long qualified naming and 2) further denote that these are jicofo-specific jingle util functions